### ISSUE TITLE
`oneOf()` creates empty string when `trim()` is used

### ISSUE THREAD

**Issue by pawelfus** [express-validator/express-validator#875]
## Example:

```js
router.post('/endpoint', [
  oneOf([
    body('reference').isString().trim().escape(),
    body('otherId').trim().escape().toInt(),
  ])
], (req, res) => {
  console.log(req.body);
});
```

Tested with body:
```js
{
  "otherId": 2
}
```

## Expected behaviour:
`req.body` should keep it's format:
```js
{
  otherId: 2
}
```

## Current behaviour:
`req.body` is modified:
```js
{
  otherId: 2,
  reference: ''
}
```

## Version
express-validator v6.4.1

## Related/similar issues:
#729

Let me know if I have missed something. For now I use custom sanitizer:
```js
const emptyString2Undefined = (value: string): string|undefined => value === '' ? undefined : value;
```

**Comment by pawelfus** [comment]
The same happens when not providing `otherId`, but `reference` only, example:

> `{ reference: "something" }`

Outputs
>  `{reference: "something", otherId: NaN }`.

**Comment by gustavohenke** [comment]
Interesting. Thanks for reporting! 🔎🐛

**Comment by gustavohenke** [comment]
A fix is out with v6.5.0! Wanna give it a try?

**Comment by pawelfus** [comment]
Thanks, that was fast! Updated and tested: works like a charm :)
